### PR TITLE
Update to 0.9.0 and rename proximus to AVS

### PR DESCRIPTION
**Pull request by davi17g**

### CODE REVIEW

**Review by jdogmcsteezy — APPROVED** [review]
There will be conflicts with my branch that will need to be resolved

File: kubernetes/manifests/avs-gke-values.yaml [hunk 1]
```diff
@@ -33,7 +33,7 @@ aerospikeVectorSearchConfig:
       - aerocluster-0-0.aerocluster.aerospike.svc.cluster.local:
```
Comment (arrowplum) [inline kubernetes/manifests/avs-gke-values.yaml:38]: The helm chart doesn't work for me with the new server. The problem is `metadata-namespace` was moved up to the `service` context https://aerospike.atlassian.net/wiki/spaces/VEC/pages/3575906306/Notable+changes+in+0.9.0+from+0.4.0#AVS-configuration

**Review by arrowplum — APPROVED** [review]
values had broken old config. Updated and added a few explicit versions. Narrowly looking at the k8s section now is working.

**Comment by jdogmcsteezy** [comment]
@davi17g I changed the base branch to VEC-218-update-1.0 and handled merge conflicts. If you have this merged into VEC-218-1.0 I can merge the final PR early tomorrow morning 8 AM PST

**Comment by davi17g** [comment]
@hev Can you please take a look?

**Review by hev — APPROVED** [review]
Just retested. Ready for merging.

**Pull request merged**